src/test/suite/extension.test.ts

```diff
@@ -20,7 +20,7 @@ suite('Extension Tests', () => {
             y
         }
         `.split('\n');
-        function f(i) {return (doc[i]); }
+        function f(i: number) { return (doc[i]); }
```

Comment:
Here on my side, eslint complains about the use of the deprecated `assert.equal`:

<img width="1151" alt="image" src="https://user-images.githubusercontent.com/4662568/100545048-cebd6780-3294-11eb-99c7-6efb68c1a625.png">

Should we do anything about it?

Reply:
Thank you for your point.
Maybe it is not eslint, it is a typescript function alert.
But It's good timing, I'll fix it to `assert.strictEqual`